mappings/net/minecraft/util/ProgressListener.mapping

```diff
@@ -1,3 +1,10 @@
 CLASS net/minecraft/class_3536 net/minecraft/util/ProgressListener
-	METHOD method_15410 progressStagePercentage (I)V
+	METHOD method_15410 setProgressPercentage (I)V
+		ARG 1 progress
 	METHOD method_15411 setDone ()V
+	METHOD method_15412 startTitledTask (Lnet/minecraft/class_2561;)V
```

Comment:
```suggestion
	METHOD method_15412 onStart (Lnet/minecraft/class_2561;)V
```
1. This method itself doesn't start a task. It's called when a task is started.
2. `title` part in the method name is unnecessary. The param name suffices.